### DISCUSSION TITLE
How to move "public" to "src/static" in "copy" plugin ?

### DISCUSSION THREAD

**Discussion by websharik** [vuejs/vue-cli#6713] [Q&A]
If "public" dir is removed - "copy" plugin not adding.

Its possible, or only add second "copy plugin" ?

I was try:
```javascript
config
.plugin('copy')
.use('CopyPlugin')
.tap(args => {
    args = [{
        from: path.resolve('../src/static'),
        to: path.resolve('../dist'),
        toType: 'dir'
    }]
    return args
})
```
but got error:
```
Error: Cannot find module 'CopyPlugin'
```

**Comment by fangbinwei** [comment]
Maybe you want to change the `args` of `copy-webpack-plugin`?  What's more, you should check the version of `copy-webpack-plugin`, so that you can confirm the structure of `args`

```js
    config
      .plugin('copy')
      .tap(args => {
        //...
        return args
      })
```

**Reply by websharik** [reply]
@fangbinwei Yep, but config.plugin('copy') is undefined if "public" dir was deleted. I think vuecli use some check  as
```
if (isDir("/public")) //add "copy" plugin...
```

**Reply by fangbinwei** [reply]
You can use your own plugin.   
 ```bash
npm install copy-webpack-plugin -D
```
```js
     config
      .plugin('my-copy')
      .use(require('copy-webpack-plugin'), [])
```
